doc/generic/pgf/CHANGELOG.md

```diff
@@ -22,8 +23,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Andreas Deininger
 - Matthias Hetzenberger
+- muzimuzhi
 - Qrrbrbirlbel
 - quark67
+- thinbold
 - Yukai Chou (muzimuzhi)
```

Comment:
I was already listed on line 27. 🤔

Reply:
Whoops, I didn't realize that. The recent update only had your nickname so I didn't even scan past the `m` …